drivers/storage/portworx/portworx.go

```diff
@@ -631,6 +631,9 @@ func (p *portworx) storageNodeToCloudSpec(storageNodes []*corev1.StorageNode, cl
 
 // SetPortworxDefaults populates default storage cluster spec values
 func SetPortworxDefaults(toUpdate *corev1.StorageCluster, k8sVersion *version.Version) {
+	if k8sVersion == nil {
```

Comment:
without this it breaks too many unit test cases. in reality this should not be nil